hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -136,7 +136,7 @@ public Response getClusterState() {
     }
 
     Long totalKeys = 0L;
-    Long deletedKeys = 0L;
+    Long keysAwaitingDeletion = 0L;
```

Comment:
Let's keep the term consistent.

```suggestion
    Long keysPendingDeletion = 0L;
```

Reply:
@smengcl @dombizita @ArafatKhan2198 Intially I used keysPendingDeletion but in review comment it was suggested to change keysAwaitingDeletion.

Reply:
@smitajoshi12 Which review comment suggested this the rename? I didn't find it.

Reply:
During a recent call with @smitajoshi12, we discussed some changes to the backend, including the naming of the response object. After some discussion, we decided to change the name to `keysAwaitingDeletion`, which I felt was more meaningful. However, I wanted to get your opinion on whether we should keep the original name, `keysPendingDeletion`, or stick with the new one. What do you think? @smengcl

Reply:
@ArafatKhan2198 On the contrary, I would think `keysAwaitingDeletion` would make contributors less experienced to Recon code more confusing. They would definitely notice the inconsistency between UI / the rest of the Ozone code (like [below](https://github.com/apache/ozone/blob/cdd96b8bafed44011d4459b65cc6192eaff7d736/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java#L608)) and this. And generally in my workflow (and maybe many other) when I'm searching the codebase with a keyword, I would expect `keypending` pointing me straight to the relevant code.

https://github.com/apache/ozone/blob/cdd96b8bafed44011d4459b65cc6192eaff7d736/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java#L608

Indeed, `keyPendingDeletion` is not the most accurate wording. But I would value code base **consistency** more in this case. We can also add comments around it.

(To be more accurate, the variable name could be `deletedKeysPendingBlockReclamation`, but don't use this because it is also inconsistent with the rest of the code base. Maybe we can do a refactoring later to change all the names consistently in one PR, or just add more comments to help other make more sense out of it. :) )

Reply:
@smengcl, I completely agree with your point on consistency. After reviewing the code snippet you posted above, I believe it would be better to revert back to the previous name we used, 'keysPendingDeletion', since we are already using something similar in the ozone code.

@smitajoshi12, I think we can revert back to the previous name!

Reply:
@smengcl  @ArafatKhan2198  @dombizita 
Hi Siyao ,
Completed revert back for keysPendingDeletion. Please  review.